packages/chrome/src/elements/header/HeaderItem.spec.tsx

```diff
@@ -63,7 +71,7 @@ describe('HeaderItem', () => {
   describe('Products', () => {
     ['chat', 'connect', 'explore', 'guide', 'message', 'support', 'talk'].forEach(product => {
       it(`renders ${product} styling if provided`, () => {
-        const { container } = render(<HeaderItem product={product} />);
+        const { container } = render(<HeaderItem product={product as PRODUCT} />);
```

Comment:
Connecting types to arrays – as done here https://github.com/zendeskgarden/react-components/blob/jzempel/grid-styles/packages/grid/src/utils/types.ts – allows us to avoid these kinds of casts.